doc/reference/tooling/tt_cli/tt_interactive_console.rst

```diff
@@ -0,0 +1,261 @@
+.. _tt-interactive-console:
+
+tt interactive console
+======================
+
+The ``tt`` utility features an command-line console that allows executing requests
+and Lua code interactively on the connected Tarantool instances.
+It is similar to the :ref:`Tarantool interactive console <interactive_console>` with
+one key difference: the ``tt`` console allows connecting to any available instance,
+both local and remote. Additionally, it offers more flexible output formatting capabilities.
+
+.. _tt-interactive-console-enter:
+
+Entering the console
+--------------------
+
+To connect to a Tarantool instance using the ``tt`` console, run :ref:`tt connect <tt-connect>`.
+Specify the instance URI and connection options, such as the username and the password,
+in the corresponding options.
+
+..  code-block:: console
+
+    $ tt connect 192.168.10.10:3301 -u myuser -p p4$$w0rD
+       • Connecting to the instance...
+       • Connected to 192.168.10.10:3301
+
+    192.168.10.10:3301>
+
+When connecting to an instance from the same ``tt`` environment, you can use the
+``<application>:<instance>`` string instead of the URI:
+
+..  code-block:: console
+
+   $ tt connect app:storage001
+       • Connecting to the instance...
+       • Connected to app:storage001
+
+    app:storage001>
+
+To get the list of supported console commands, enter ``\help`` or ``?``.
+To quit the console, enter ``\quit`` or ``\q``.
+
+.. _tt-interactive-console-input:
+
+Console input
+-------------
+
+Similarly to the :ref:`Tarantool interactive console <interactive_console>`, the
+``tt`` console can handle Lua or SQL input. The default is Lua. To change the input
+language, run ``\set language <language>``, for example:
+
+..  code-block:: console
+
+    app:storage001> \set language sql
+    app:storage001> select * from bands where id = 1
+    ---
+    - metadata:
+      - name: id
+        type: unsigned
+      - name: band_name
+        type: string
+      - name: year
+        type: unsigned
+      rows:
+      - [1, 'Roxette', 1986]
+    ...
+
+..  code-block:: console
+
+    app:storage001> \set language lua
+    app:storage001> box.space.bands:select { 1 }
+    ---
+    - - [1, 'Roxette', 1986]
+    ...
+
+.. note::
+
+    You can also specify the input language in the ``tt connect`` call using the
+    ``-l``/``--language`` option:
+
+    ..  code-block:: console
+
+        $ tt connect app:storage001 -l sql
+
+For Lua input, the tab-based autocompletion works automatically for loaded modules.
+
+.. _tt-interactive-console-output:
+
+Console output
+--------------
+
+By default, the ``tt`` console prints the output data in the YAML format, each
+tuple on the new line:
+
+..  code-block:: console
+
+    app:storage001> box.space.bands:select { }
+    ---
+    - - [1, 'Roxette', 1986]
+      - [2, 'Scorpions', 1965]
+      - [3, 'Ace of Base', 1987]
+    ...
+
+You can switch to alternative output formats -- Lua or human-readable tables --
```

Comment:
Don't know why but `human-readable` sounds a bit weird to me in this context :) YAML also looks pretty human-readable